whereby/sdk
# common: Turn remote camera off/ask to turn camera on

### Description

Add the possibility for hosts to turn off cameras for remote participants. The action turnOffParticipantCameras takes an array of clientIds, so you can also implement "turn off all cameras" functionality with this. Also adds `askToTurnOnCamera` function, that a host can use to ask a participant to enable their camera.

**Summary:**



**Related Issue:**



### Testing

1. Check out this branch
2. In the /.env file add a valid host roomKey for the configured STORYBOOK_ROOM in following line:
STORYBOOK_ROOM_HOST_ROOMKEY=[host_roomKey_here]
3. Run yarn build && yarn dev
4. In the Storybook UI (@ http://localhost:6006/) open the story: "Custom UI" > "Room Connection With Host Controls"
5. Join the same room in a different browser (PWA).
6. Clicking on the turn off camera button on the remote participant should turn the camera off in the other tab (PWA side). 
![image](https://github.com/user-attachments/assets/ee5b9923-746f-4442-9f7d-306cc2818400)
7. Verify that it in fact does turn the camera off, you should see this notification in the PWA tab:
![image](https://github.com/user-attachments/assets/eb46451d-97fe-4de5-9513-938d4101acdf)
8. Test the other direction -> verify that turning cam off from the PWA does turn off in the SDK (and a notification will be shown in storybook).
9. Additionally, this can also be tested as a non-host (remove the host key from .env for example) - the actions should not work. 

### Screenshots/GIFs (if applicable)



### Checklist

        under the terms of the MIT license.

### Additional Information
